nucypher/cli/commands/deploy.py

```diff
@@ -257,14 +264,19 @@ def download_registry(general_config, config_root, registry_outfile, network, fo
 @option_deployer_address
 @option_poa
 @option_ignore_solidity_version
-def inspect(general_config, provider_uri, config_root, registry_infile, deployer_address, poa, ignore_solidity_check):
+@option_gas_strategy
+def inspect(general_config, provider_uri, config_root, registry_infile, deployer_address, poa, ignore_solidity_check, gas_strategy):
```

Comment:
Gas strategy doesn't have much sense here, since `inspect` is a transactionless command.

Reply:
:heavy_check_mark: